.github/workflows/trusted_sync_docker.yaml

```diff
@@ -34,5 +39,5 @@ jobs:
           file: examples/trusted-sync/Dockerfile
           context: .
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: ${{ steps.meta.outputs.tags }},ghcr.io/ethereum-optimism/kona/trusted-sync:${{ github.event.inputs.customTag }}
```

Comment:
Think we don't need the full path, yeah? Should just need `GitHub.event.inputs.customTag`